python/src/trezorlib/firmware/nrf.py

```
@@ -0,0 +1,303 @@
+
+import construct as c
+from construct_classes import subcon
+from typing_extensions import Self
+
+from .. import _ed25519 as ed25519
+from ..construct_helpers import EnumAdapter, TupleAdapter
+from . import util
+from .models import Model, get_nrf_keys
+from .sanity_struct import STRICT_SANITY_CHECK_DEFAULT, SanityCheckedStruct
+
+__all__ = ["NrfImage", "NrfHeader"]
```

Ruff (0.15.18)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)